src/test/java/app/gpx_animator/KmHrMinPerKmTest.java

```diff
@@ -0,0 +1,58 @@
+package app.gpx_animator;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public class KmHrMinPerKmTest {
+
+    private String calcKmPerHrToMinPerKm(final long speed) {
+        double minPerKm = speed;
+        double mins = (60 / minPerKm);
+        double secs = (mins - (int) mins) * 60;
+        double tenS = 10;
+        if (minPerKm != 0) {
+            if (secs == 0) {
+                return String.format("%d km/h | " + (int) mins + ":00 minutes/km", speed); //NON-NLS
+            } else if (secs >= tenS) {
+                return String.format("%d km/h | " + (int) mins + ":" + (int) secs + " minutes/km", speed); //NON-NLS
+            } else {
+                return String.format("%d km/h | " + (int) mins + ":0" + (int) secs + " minutes/km", speed); //NON-NLS
+            }
+        } else {
+            return String.format("%d km/h | 0:00 minutes/km", speed); //NON-NLS
+        }
+    }
```

Comment:
This method looks identical to the same method in the ´Renderer´ class. In this case you test the method in the test class, not the method in the ´Renderer´ class.